src/pretix/static/pretixbase/scss/_theme.scss

```
@@ -225,6 +225,10 @@ svg.svg-icon {
   top: 2px;
 }
 
+svg.svg-icon.text-muted path {
```

Ich bin mir nicht sicher, ob es nicht vielleicht flexibler wäre, den Selektor zu ändern, so dass auch SVGs innerhalb eines `.text-muted` Elements entsprechend grau dargestellt werden. Das fill im CSS sollte ja eigentlich nur überschreiben, wenn im eigentlichen SVG nichts anderes angegeben ist.

```suggestion
.text-muted path {
```

---

Ist mir jetzt bisschen spekulativ, was für SVGs das treffen sollte, die sich nicht wie icons verhalten sollen…